agent/utils/fips_windows.go

```diff
@@ -0,0 +1,28 @@
+//go:build windows
```

Comment:
do we also need a no-op method in an unsupported file? like https://github.com/aws/amazon-ecs-agent/blob/master/agent/utils/utils_unsupported.go#L1-L2

Reply:
+1 an "unsupported" file with build constraint `!linux` would probably be better here so that we exclude macos and others correctly